### DISCUSSION TITLE
Istallazione giuaschool

### DISCUSSION THREAD

**Discussion by dirigenteic-demo** [iisgiua/giuaschool#522] [Richieste di aiuto]
Salve vorrei avere il tutorial per istallare giuaschool, non mi ricordo bene i passaggi. 
Cordiali saluti
Prof.ssa Michela Pignatelli

**Comment by trinko** [comment]
La documentazione è tutta qui: [installazione](https://iisgiua.github.io/giuaschool-docs/install.html)
Altrimenti suggerisco di contattore chi si è offerto di fare l'installazione: [vedi annuncio](https://github.com/iisgiua/giuaschool/discussions/504)

**Comment by fabriziotoppi** [comment]
guarda io ho installato la scorsa settimana il registro elettronico . posso aitarti se vuoi,

**Reply by mpignatelli** [reply]
Ciao mi potresti aiutare con l'istallazione grazie mille

**Comment by fabriziotoppi** [comment]
Certo mandami la tua mail

Il sab 29 lug 2023, 12:15 mpignatelli ***@***.***> ha scritto:

**Reply by mpignatelli** [reply]
Ecco:michelina.pignatelli1983@gmail.com

**Reply by CobraOfficial01** [reply]
potresti aiutare pure a me

**Reply by dannycastellano** [reply]
Riusciresti ad aiutare anche me? Ti ringrazio in anticipo.

**Comment by fabriziotoppi** [comment]
CERTO


Il giorno lun 18 mar 2024 alle ore 10:48 Dany ***@***.***> ha
scritto:

**Reply by CobraOfficial01** [reply]
ok grazie

**Comment by CobraOfficial01** [comment]
come facciamo a sentirci per l'installazione

**Comment by CobraOfficial01** [comment]
la mia mail è danyventuri09@gmail.com

**Comment by fabriziotoppi** [comment]
Hai già un hosting m?

Il lun 18 mar 2024, 15:10 Danny Castellano ***@***.***> ha
scritto:

**Reply by dannycastellano** [reply]
si

**Reply by CobraOfficial01** [reply]
no non ho un hosting

**Comment by fabriziotoppi** [comment]
Mi servono i dati  per poter entrare . Altervista?

Il lun 18 mar 2024, 15:29 Danny Castellano ***@***.***> ha
scritto:

**Reply by dannycastellano** [reply]
@fabriziotoppi 
Si Altervista, dove ti mando le credenziali?

**Comment by fabriziotoppi** [comment]
***@***.***
Nn prima.di domani però

Il lun 18 mar 2024, 15:42 Danny Castellano ***@***.***> ha
scritto:

**Reply by dannycastellano** [reply]
Ok va bene, nessun problema, se mi lasci una mail ti mando le credenziali.

**Reply by CobraOfficial01** [reply]
ok va bene

**Comment by CobraOfficial01** [comment]
@fabriziotoppi mi contatti
